src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Figure;
+import core.basesyntax.randomizer.FigureSupplier;
+
+public class Main {
+    public static void main(String[] args) {
+        final int lengthOfArray = 6;
```

Comment:
Please, pay more attention to what constant variable means in context of Java. 
Here is [hint](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants).

Reply:
I have a question, when I write "final int ARRAY_LENGTH" maven is failing build, why? Hint says that I need to name constants like "public static final int FIGURE_COUNT = 5;".